pkg/cmd/gist/list/list.go

```diff
@@ -119,13 +193,103 @@ func listRun(opts *ListOptions) error {
 		tp.AddField(gist.ID)
 		tp.AddField(
 			text.RemoveExcessiveWhitespace(description),
-			tableprinter.WithColor(cs.Bold),
+			tableprinter.WithColor(highlightDescription),
+		)
+		tp.AddField(
+			text.Pluralize(fileCount, "file"),
+			tableprinter.WithColor(highlightFilesFunc(&gist)),
 		)
-		tp.AddField(text.Pluralize(fileCount, "file"))
 		tp.AddField(visibility, tableprinter.WithColor(visColor))
 		tp.AddTimeField(time.Now(), gist.UpdatedAt, cs.Gray)
 		tp.EndRow()
 	}
 
 	return tp.Render()
 }
+
+func printContent(io *iostreams.IOStreams, gists []shared.Gist, filter *regexp.Regexp) error {
```

Comment:
Since this is new, should we document what the output is expected to look like in some way?

Reply:
You mean like a doc comment or in user-facing docs?

Reply:
🤔 yes? But a doc comment here is probably good enough. Mostly so I can see "this function should print something of this format"